pkg/operator/controller/garden/garden/components.go

```diff
@@ -1156,7 +1156,9 @@ func (r *Reconciler) newBlackboxExporter(garden *operatorv1alpha1.Garden, secret
 			VPAEnabled:        true,
 			KubernetesVersion: r.RuntimeVersion,
 			PodLabels: map[string]string{
-				gardenerutils.NetworkPolicyLabel(gardenerapiserver.DeploymentName, 8443): v1beta1constants.LabelNetworkPolicyAllowed,
+				v1beta1constants.LabelNetworkPolicyToPublicNetworks: v1beta1constants.LabelNetworkPolicyAllowed,
+				"networking.resources.gardener.cloud/to-" + v1beta1constants.LabelNetworkPolicyIstioIngressNamespaceAlias + "-" + v1beta1constants.DefaultSNIIngressServiceName: v1beta1constants.LabelNetworkPolicyAllowed,
```

Comment:
```suggestion
				gardenerutils.NetworkPolicyLabel(v1beta1constants.LabelNetworkPolicyIstioIngressNamespaceAlias+"-"+v1beta1constants.DefaultSNIIngressServiceName, 9443)
```
Otherwise `-tcp-9443` would be missing, at least according to my history from yesterday 😄  
```
 k get networkpolicies -n garden egress-to-virtual-garden-istio-ingress-istio-ingressgateway-tcp-9443 -o yaml
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  annotations:
    gardener.cloud/description: Allows egress TCP traffic to port 9443 from pods running
      in namespace garden labeled with {map[networking.resources.gardener.cloud/to-all-istio-ingresses-istio-ingressgateway-tcp-9443:allowed]
      []} to pods selected by the virtual-garden-istio-ingress/istio-ingressgateway
      service selector.
  creationTimestamp: "2024-04-10T10:43:01Z"
  generation: 1
  labels:
    networking.resources.gardener.cloud/service-name: istio-ingressgateway
    networking.resources.gardener.cloud/service-namespace: virtual-garden-istio-ingress
  name: egress-to-virtual-garden-istio-ingress-istio-ingressgateway-tcp-9443
  namespace: garden
  resourceVersion: "2355"
  uid: 362aee97-39c9-4817-854a-0acc97810575
spec:
  egress:
  - ports:
    - port: 9443
      protocol: TCP
    to:
    - namespaceSelector:
        matchLabels:
          kubernetes.io/metadata.name: virtual-garden-istio-ingress
      podSelector:
        matchLabels:
          app: istio-ingressgateway
          istio: ingressgateway
  podSelector:
    matchLabels:
      networking.resources.gardener.cloud/to-all-istio-ingresses-istio-ingressgateway-tcp-9443: allowed
  policyTypes:
  - Egress
```

Reply:
... :) Thank you